baus/ual.py

```diff
@@ -705,8 +705,8 @@ def save_intermediate_tables(households, buildings, parcels,
 # have to define this here because urbansim_defaults incorrectly calls the
 # outcome variable non_residential_price
 @orca.step('nrh_simulate')
-def nrh_simulate(buildings, aggregations, nrh_config):
-    return utils.hedonic_simulate(nrh_config, buildings, aggregations,
+def nrh_simulate(buildings, aggregations):
```

Comment:
`rsh_config` was previously specified in `inputs.yaml`; see the "PR Overview" for why that yaml was removed. Since it was removed the direct file name was added back. This is related to our other discussion about creating variables rather than hard-coding file names, but that specific yaml made it a bit harder to understand which files were in use (though probably largely because it had another complication where if you forgot to update the file name it would use the wrong file). Anyway, it sounds like we'll decide about hardcoding via the [Asana task](https://app.asana.com/0/385259290425521/1203767459263968).